zkevm-circuits/src/pi_circuit.rs

```diff
@@ -518,6 +570,32 @@ impl<F: Field> SubCircuitConfig<F> for PiCircuitConfig<F> {
             },
         );
 
+        // 2b. constrain rpi_rlc and rpi_length_acc within chunk_txbytes section
+        meta.create_gate(
```

Comment:
You also need to constrain `rpi_length_acc`.

For non-padded rows, the lookup to the TxTable handles it, but you need to check that for padded rows, the `rpi_length_acc` does not change.

You might also need to check that the `rpi_length_acc` at the _first row_ of the `q_chunk_txbytes` section is correct? The lookup checks the length differential, so I believe you must be sure that the `rpi_length_acc` on the previous row is `0` (but may be this is already true because of the `init` row). Please check though

Reply:
`rpi_length_acc` consistency constraint for padding rows is added in this patch: https://github.com/scroll-tech/zkevm-circuits/pull/1180. 

The correctness of `rpi_length_acc` at the first row is indeed included by the TxTable lookup as the `chunk_txbytes` section also has the `init` row as a padding with `rpi_length_acc` being 0.